### DISCUSSION TITLE
Loosing Wlan after some time with latest Version

### DISCUSSION THREAD

**Discussion by hjfisch** [rern/rAudio#984] [Bug Reports]
It only happens with internal RPI 4 Wlan , not with USB-Wlan-Stick .
Back to  202207101 for now .

**Comment by hjfisch** [comment]
Still happends with latest Version.

**Reply by rern** [reply]
Please try:
Addons > rAudio > long-press `Update` - `OK` for branch `UPDATE`

**Comment by hjfisch** [comment]
I'll try it , thank you .
But there is still the Samba problem .
After a while the Samba - Connection won't work anymore .
Need to connect with putty and "systemctl enable smb nmb".
After doing so , Samba is rock solid.

**Comment by hjfisch** [comment]
No Wlan again today.
Never happends with 202207101 .
Runs one or two days .

**Comment by rern** [comment]
Pleas try:
```sh
iw wlan0 get power_save
```
If the result is `Power save: on`, reboot and the issue should be resolved.

**Comment by hjfisch** [comment]
Thank's ,
Wlan Power_Save was Off by default .
Set it to On now .

**Reply by rern** [reply]
Hold on.
It must be set to off and it's off by default. Otherwise it will go to sleep and drop the connection.

**Comment by hjfisch** [comment]
Setting it back to off again.
To make it clear , Wlan drops from time to time .Sometimes after an hour an sometimes after 2 days . I've 3 systems running and it
only happends with internal wlan . 2 of my  sytems are running with external USB-Wlan-Adapters . These 2 are never droping the wlan connection. With Ver.: 202207101 internal Wlan is running rock solid .
B.t.w. my systems are running 24/7 because they are build into my active speakers with digital inputs.
